vpc-endpoints.tf

```
@@ -62,7 +62,7 @@ resource "aws_security_group" "logs_interface_endpoint" {
     from_port   = 443
     to_port     = 443
     protocol    = "TCP"
-    cidr_blocks = [var.compute_subnet_cidr_block, var.application_subnet_cidr_block]
+    cidr_blocks = [var.compute_subnet_cidr_block, var.application_subnet_cidr_block, var.data_subnet_cidr_blocks[0], var.data_subnet_cidr_blocks[1]]
```

Wouldn't we need to include every item in the `data_subnet_cidr_blocks` and `data_subnet_cidr_blocks` lists?

```suggestion
    cidr_blocks = concat([var.compute_subnet_cidr_block, var.application_subnet_cidr_block], var.data_subnet_cidr_blocks, var.data_subnet_cidr_blocks)
```

---

Thanks for your suggestion Dan. I updated it. Thanks,